src/components/MapView/DateSelector/index.tsx

```diff
@@ -177,7 +178,7 @@ const styles = (theme: Theme) =>
   });
 
 export interface DateSelectorProps extends WithStyles<typeof styles> {
-  availableDates: number[];
+  availableDates?: number[];
```

Comment:
What happens to `includeDates={availableDates.map(d => new Date(d + userOffset))}` if availableDates is undefined?

Reply:
the value is default set to an empty array here:
https://github.com/WFP-VAM/prism-frontend/blob/cad0637bd405c3d4d166d0ca7bc098efcb9711f1/src/components/MapView/DateSelector/index.tsx#L39
The reason I changed the type is because the default array setting was already there, so I updated the type definition to reflect this. Mostly to prevent an error here: 
https://github.com/WFP-VAM/prism-frontend/blame/ca955c5cc5d3088d591b0d05719be0f604116231/src/components/MapView/DateSelector/index.test.tsx#L18
but the Analysis PR #92 seems to have fixed it by just giving an empty array instead.

Which do you think is the best approach?
1. Revert this change and use what was done in #92 (passing empty array in test)
2. Use what was done here and rely on default if availableDates isn't given, and type the props accordingly

Reply:
Ok I think that's propably like that then

Reply:
I'll leave it as it is if that's what you mean 🙂